pkg/accounting/accounting.go

```diff
@@ -999,7 +1005,8 @@ func (a *Accounting) NotifyPaymentSent(peer swarm.Address, amount *big.Int, rece
 
 	err = a.store.Put(peerBalanceKey(peer), nextBalance)
 	if err != nil {
-		a.logger.Error(err, "notify payment sent; failed to persist balance")
+		a.logger.Debug("notify payment sent; failed to persist balance", "error", err)
+		a.logger.Error(nil, "notify payment sent; failed to persist balance")
```

Comment:
Another redundant logger.Debug if you put the err in the logger.Error.

Reply:
comment added below